tests/core/test_pandas_engine.py

```diff
@@ -14,9 +15,20 @@
 from pandera.engines import pandas_engine
 from pandera.errors import ParserError
 
+UNSUPPORTED_DTYPE_CLS: set[Any] = set()
```

Comment:
use `typing.Set` here for compat with py3.8

Reply:
oops that was a force of habit, let me fix it